src/core/config/descriptor_providers/xlio_config_schema.json

```diff
@@ -0,0 +1,1454 @@
+                    "default": 32,
+                    "title": "Maximum GRO streams",
+                    "description": "Control the number of TCP streams to perform Generic Receive Offload simultaneously. Maps to XLIO_GRO_STREAMS_MAX environment variable. Disable GRO with a value of 0."
+                }
+            }
+        },
+        "applications": {
+            "type": "object",
+            "title": "Application Integration",
+            "description": "Configurations and optimizations for specific applications or use cases.",
+            "properties": {
+                "nginx": {
+                    "type": "object",
+                    "description": "NGINX-specific optimizations.",
+                    "properties": {
+                        "src_port_stride": {
+                            "type": "integer",
+                            "default": 2,
+                            "title": "Source port stride",
+                            "description": "Controls how source ports are distributed across Nginx worker processes. Maps to XLIO_NGINX_SRC_PORT_STRIDE environment variable."
+                        },
+                        "workers_num": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "Number of workers",
+                            "description": "Number of Nginx worker processes to optimize for. This parameter must be set to offload Nginx. Maps to XLIO_NGINX_WORKERS_NUM environment variable."
+                        },
+                        "udp_pool_size": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "UDP buffer pool size",
+                            "description": "The size of UDP socket pool for NGINX. Maps to XLIO_NGINX_UDP_POOL_SIZE environment variable. For any value different than 0 - close() socket will not destroy the socket, but will place it in a pool for next socket UDP creation.\nDisable with 0"
+                        },
+                        "udp_socket_pool_reuse": {
+                            "type": "boolean",
+                            "default": false,
+                            "title": "Enable UDP socket pool reuse",
+                            "description": "Allows reuse of UDP socket pools for NGINX deployments. Maps to XLIO_NGINX_UDP_POOL_RX_NUM_BUFFS_REUSE environment variable."
+                        },
+                        "distribute_cq": {
+                            "type": "boolean",
+                            "default": false,
+                            "title": "Distribute completion queues",
+                            "description": "Distributes completion queue processing across worker processes for better performance. Maps to XLIO_DISTRIBUTE_CQ environment variable."
+                        }
+                    }
+                },
+                "profiles": {
```

Comment:
i would put this as seperate global section or under performance as these should be the entry point of xlio configuration